astroquery/jplhorizons/tests/test_jplhorizons_remote.py

```diff
@@ -284,6 +284,22 @@ def test_vectors_query_raw(self):
 
         assert len(res) >= 6412
 
+    def test_vectors_query_topocentric_coordinates(self):
+        "Test vectors query specifying observer's longitude, latitude, and elevation"
+        q = jplhorizons.Horizons(id='Ceres',
+                                 location={"lon": 244, "lat": 33, "elevation": 1700},
+                                 id_type='smallbody',
+                                 epochs=2451544.5)
+        res = q.vectors_async()
```

Comment:
This seems more to cover the status quo use case, to ensure the topocentric coordinates are used in the query. Could you maybe add another test function where you indeed use `Quantity` and also preferably an `EarthLocation()` as an input for the `location` kwarg?. Alternatively it could be just put into a `pytest.mark.paramertise along with the current dictionary.

Reply:
Thanks for that suggestion.  I've added a test with units.  Regarding `EarthLocation`, I would like to have that added, but I won't be able to do so until... late September?

Reply:
(added, as in, the Horizons code does not support `EarthLocation`)

Reply:
ah, OK, that's fine, and certainly doesn't have to be part of this PR.